packages/wallets/src/components/WalletsCarousel/WalletsCarousel.scss

```diff
@@ -0,0 +1,3 @@
+.wallets-carousel {
+    width: 100vw;
```

Comment:
Is not common for a child container to have `100vw` can we use `100%` instead and make the parent `100vw`?

Reply:
Updated